### DISCUSSION TITLE
Implement offline storage for PWA

### DISCUSSION THREAD

**Discussion by AliMickey** [photoprism/photoprism#1993] [Q&A] [answered]
If the container is restarted, all clients need to log in again as the session is wiped. 

Implement offline storage for session data.

**Comment by lastzero** [comment, marked as answer]
This should not be the case unless your storage folder isn't writable or you cleared your browser cache / used the private browser mode... Did you check your logs already?

**Reply by AliMickey** [reply]
It seems that the UID/GID environment variables had changed names. Once I added the `PHOTOPRISM_` prefix, it is working now.


Thanks
